app/Helpers/database/ticket.php

```diff
@@ -88,76 +89,93 @@ function submitNewTicket(User $user, int $achID, int $reportType, int $hardcore,
     return _createTicket($user, $achID, $reportType, $hardcore, $note);
 }
 
-function _createTicket(User $user, int $achID, int $reportType, ?int $hardcore, string $note): int
-{
-    $achievement = Achievement::find($achID);
-    if (!$achievement) {
-        return 0;
-    }
-
-    $noteSanitized = $note;
-    sanitize_sql_inputs($noteSanitized);
-
-    $hardcoreValue = $hardcore === null ? 'NULL' : (string) $hardcore;
+function constructAchievementTicketBugReportDetails(
+    Ticket $ticket,
+    Game $game,
+    Achievement $achievement
+): string {
+    $problemTypeStr = ($ticket->type === 1) ? "Triggers at wrong time" : "Doesn't trigger";
+    $ticketUrl = route('ticket.show', ['ticket' => $ticket]);
 
-    $userId = $user->id;
-    $username = $user->User;
-
-    $query = "INSERT INTO Ticket (AchievementID, reporter_id, ReportType, Hardcore, ReportNotes, ReportedAt, ResolvedAt, resolver_id )
-              VALUES($achID, $userId, $reportType, $hardcoreValue, \"$noteSanitized\", NOW(), NULL, NULL )";
-
-    $db = getMysqliConnection();
-    $dbResult = mysqli_query($db, $query);
-    if (!$dbResult) {
-        log_sql_fail();
-
-        return 0;
-    }
-
-    $ticketID = mysqli_insert_id($db);
-
-    $achTitle = $achievement->title;
-    $gameID = $achievement->game->id;
-    $gameTitle = $achievement->game->title;
-
-    expireUserTicketCounts($achievement->developer->User);
-
-    $problemTypeStr = ($reportType === 1) ? "Triggers at wrong time" : "Doesn't trigger";
-
-    $emailHeader = "Bug Report ($gameTitle)";
-    $ticketUrl = route('ticket.show', ['ticket' => $ticketID]);
     $bugReportDetails = "
-Achievement: $achTitle
-Game: $gameTitle
-Problem: $problemTypeStr
-Comment: $note
+Achievement: {$achievement->title}
+Game: {$game->title}
+Problem: {$problemTypeStr}
+Comment: {$ticket->body}
```

Comment:
I tested creating a new ticket and the Comment did not show up in the email.

> Jamiras would like to report a bug with an achievement you've created:
>
> Achievement: Who Let the Dogs Out
> Game: Legend of Zelda, The: Ocarina of Time
> Problem: Doesn't trigger
> Comment:
>
> This ticket will be raised and will be available for all developers to inspect and manage at the following URL:
http://localhost:64000/ticket/72901

Reply:
Good catch - looks like I missed checking in a model update. Updated in latest.